projects/canopy/src/lib/banner/docs/banner.stories.ts

```diff
@@ -0,0 +1,111 @@
+import { Component, Input } from '@angular/core';
+import { Meta, moduleMetadata, Story } from '@storybook/angular';
+
+import { LgIconRegistry } from '../../icon/icon.registry';
+import { LgIconModule, lgIconsArray } from '../../icon';
+import { LgBannerModule } from '../banner.module';
+import { LgBannerComponent } from '../banner.component';
+
+const variantTypes = [ 'generic', 'warning' ];
+
+@Component({
+  selector: 'lg-banner-icon',
+  template: `
+    <lg-banner [variant]="variant">
+      <lg-icon [name]="icon"></lg-icon>
+      {{ content }}
+    </lg-banner>
+  `,
+})
+class LgBannerIconComponent {
+  @Input() content: string;
+  @Input() variant: string;
+  @Input() icon: string;
+
+  constructor(private registry: LgIconRegistry) {
+    this.registry.registerIcons(lgIconsArray);
+  }
+}
+
+// This default export determines where your story goes in the story list
+export default {
+  title: 'Components/Banner message/Examples',
```

Comment:
Could this be `'Components/Banner message (WIP)/Examples'` while it's a work in progress please?